pool/pool_server.py

```diff
@@ -48,12 +51,27 @@ def check_authentication_token(launcher_id: bytes32, token: uint64, timeout: uin
     return None
 
 
+def get_ssl_context(config):
+    if config["server"]["server_use_ssl"] is False:
+        return None
+    ssl_context = ssl.create_default_context(ssl.Purpose.CLIENT_AUTH)
+    ssl_context.load_cert_chain(config["server"]["server_ssl_crt"], config["server"]["server_ssl_key"])
+    return ssl_context
+
+
 class PoolServer:
     def __init__(self, config: Dict, constants: ConsensusConstants, pool_store: Optional[AbstractPoolStore] = None):
 
         self.log = logging.getLogger(__name__)
         self.pool = Pool(config, constants, pool_store)
 
+        with open(os.getcwd() + "/config.yaml") as f:
```

Comment:
Hm, there are two places now where this happens, lets wrap it or pass it in here?

Reply:
good suggestion, pass it in here.